src/store/index.js

```diff
@@ -29,64 +29,46 @@ export default createStore({
         id: 1,
         name: "Nike Air Force 1",
         img: "NikeAirForce1",
-        price: 20,
-        inStock: false,
       },
       {
         id: 2,
         name: "Nike Air Max 2021",
         img: "NikeAirMax2021",
-        price: 20,
-        inStock: false,
       },
       {
         id: 3,
         name: "Nike Air Max 270",
         img: "NikeAirMax270",
-        price: 20,
-        inStock: false,
       },
       {
         id: 4,
         name: "Nike Air Max 97",
         img: "NikeAirMax97",
-        price: 20,
-        inStock: false,
       },
       {
         id: 5,
         name: "Nike Air Max Genome",
         img: "NikeAirMaxGenome",
-        price: 20,
-        inStock: false,
       },
       {
         id: 6,
         name: "Nike Air Presto",
         img: "NikeAirPresto",
-        price: 20,
-        inStock: false,
       },
       {
         id: 7,
         name: "Nike Air Zoom 37",
         img: "NikeAirZoomPegasus37",
-        price: 20,
-        inStock: false,
       },
       {
         id: 8,
         name: "Nike And Rtfkt",
         img: "NikeAndRtfkt",
-        price: 20,
-        inStock: false,
       },
       {
         id: 9,
         name: "Nike Revolution 6",
         img: "NikeRevolution6",
-        price: 20,
-        inStock: false,
```

Comment:
Tutaj będziemy też to ogarniać za pomocą ajaxa. Ja Ci wystawię jakieś api - więc to możesz olać